content/en/aws/kms/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Key Management Service (KMS)"
+linkTitle: "Key Management Service (KMS)"
+categories: ["LocalStack Community"]
+description: >
+  Key Management Service (KMS)
+---
+
+Key Management Service (KMS) is mainly about
+- creation and management of cryptographic keys
+- use of these keys to encrypt / decrypt data or to sign messages and to verify signatures for signed ones
```

Comment:
Please use full sentences rather than points in the initial introduction. It would also be recommended to use camel case for writing sentences.